### ISSUE TITLE
RalliSport Challenge 2: Few crashes towards the end of the Pro championship

### ISSUE THREAD

**Issue by Valtekken** [xemu-project/xemu#1378]
### Title

https://xemu.app/titles/4d530039/#RalliSport-Challenge-2

MS-057

### Bug Description

This only happened a few times and only one day (I played the entirety of the game's campaign over the course of 20 hours or so, daily sessions), so it could've been a temporary issue that didn't specifically depend on Xemu or on the game, but I'll file it anyway under @mborgerson 's suggestion.
The last few stages of the Pro championship crashed on me 2 or 3 times within the span of an hour or so. The crash basically causes a freeze of the current frame on screen and causes the game audio to loop. Clicking on the window a few times prompts Windows to tell the user the app is non-responsive and has to be forcibly shutdown.

### Expected Behavior

No crashes whatsoever.

### xemu Version

0.7.83

### System Information

OS: Windows 11 22H2
CPU: AMD Ryzen 5 5600X
GPU: NVidia GeForce GTX 970
GPU Driver: 4.0.0 NVIDIA 528.24

**Comment by LeRutY** [comment]
I have crashes in multiplayer too (usually after 30 minutes of playing), it´s instability of the emulator, RalliSport Challenge 2 isn´t the only game which is affected

**Comment by Valtekken** [comment]
> I have crashes in multiplayer too (usually after 30 minutes of playing), it´s instability of the emulator, RalliSport Challenge 2 isn´t the only game which is affected

Yeah, I've been told a probable cause has been found. Maybe it's a common cause.

**Comment by mattbillpage** [comment]
I want to add onto this to say that these crashes may be an issue with the game itself; Running on original hardware I've experienced crashes towards the end of the Pro championship aswell; I havent tested on multiplayer after 30 minutes of playing
